app/docs/setup.md

```diff
@@ -0,0 +1,28 @@
+
+The project expects a gradle-variable defined inside `~/.gradle/gradle.properties`,
+which points to the secrets-directory for this project.
+
+`ANIMEAL_SECRETS_FOLDER=/Users/Janis_Graumanis/Documents/secrets/animeal-app-secrets`
+
+At this point the project expects that directory to contain the two config-files generated by aws-amplify:
+* awsconfiguration.json
+* amplifyconfiguration.json
+
+
+
+
+Follow instructions here for preparation required to generate your own copies of those config-json files:
+https://kb.epam.com/display/EPMEDU/Access+AWS+resources
```

Comment:
Can we remove links related to epam?

Reply:
@yaanis-epam We have to avoid refers to any epam resources in this app since it's a volunteer app.
As an option we can add the link to Android page on KB: https://kb.epam.com/display/EPMEDU/%5BEPM-ANML%5D+%5BAndroid%5D+Development

Reply:
It seems like the correct solution for this would be to move away from using epam-kb & epam-jira; the project does not truly feel open-source/volunteer until those become accessible to outsiders.